README.md

```diff
@@ -2,78 +2,26 @@
-
-# Where to contribute
-
-There are 2 directories where you can contribute, depending on a set of requirements:
-
-| Target Directory               | Type of Operators              | Target Platform             | Requirements                                                  |
-|--------------------------------|--------------------------------|-----------------------------|---------------------------------------------------------------|
-| `community-operators`          | Community OpenShift Operators  | OpenShift / OKD             | needs to work on OpenShift 3.11 or newer                      |
-| `upstream-community-operators` | Community Kubernetes Operators | Kubernetes / OperatorHub.io | needs to work on Kubernetes 1.7 or newer                      |
-
-The column _Target Platform_ denotes where this Operator will be visible (embedded OperatorHub in OpenShift and OKD, or OperatorHub.io for Kubernetes) and where it's intended to run.
-
-For partners and ISVs, certified operators can now be submitted via connect.redhat.com
-
-**If you Operator works on both Kubernetes and OpenShift, place a copy of your bundle in the `upstream-community-operators` directory, as well as the `community-operators` directory.**
-
-Note that OpenShift and OKD clusters by default come with access to operators from `community-operators`, certified operators, and additional Red Hat operators. Please keep this in mind when submitting operators, as duplicate operators between these sources will not be tolerated. 
-
-# Before submitting a PR
+Similarly, to update your operator you need to submit a PR with any changes to your Operator resources. Refere to our [contribution guide](docs/contributing.md#updating-your-existing-operator) for more details.
 
 ## Test your Operator
 
-Upon creating a pull request against this repo, a set of CI pipelines will run, see more details [here](https://github.com/operator-framework/community-operators/blob/master/docs/ci.md).
-
-You can help speed up the review of your PR by [testing manually](https://github.com/operator-framework/community-operators/blob/master/docs/testing-operators.md).
-
-## Verify your Operator Metadata
+Upon creating a pull request against this repo, a set of CI pipelines will run, see more details [here](./docs/ci.md).
 
-The maintainers will work with you to make sure your Operator has the required metadata to function properly and be displayed with useful information for the end user.
-
-You can help us with that by validating your `bundle` with [operator-courier](https://github.com/operator-framework/operator-courier). This tool will check against the [required fields][required-fields] in your CSV.
-
-```sh
-operator-courier verify --ui_validate_io path/to/bundle
-```
+You can help speed up the review of your PR by [testing locally](./docs/testing-operators.md).
```

Comment:
Given you are revamping the doc, I suggest changing line 23 and say `using the preview tool` and embedding the link in `preview tool`